SPECS/openssl/openssl.spec

```
@@ -365,10 +367,13 @@ rm -f %{buildroot}%{_sysconfdir}/pki/tls/ct_log_list.cnf.dist
 %postun libs -p /sbin/ldconfig
 
 %changelog
+* Thu May 23 2024 Juan Camposeco <juan.camposeco@gmail.com> - 1.1.1k-32
+ - Implicit rejection of PKCS#1 v1.5 (CVE-2023-50782) - cherrypick from fasttrack/2.0
+
 * Tue Jun 04 2024 Tobias Brick <tobiasb@microsoft.com> - 1.1.1k-31
 - Only free the read buffers if we're not using them
 
-* Fri Apr 19 2024 Tobias Brick <tobiasb@microsoft.com> - 1.1.1k-30
+* Fri Apr 19 2024 Tobias Brick <tobaisb@microsoft.com> - 1.1.1k-30
```

I had fixed a typo here, which this accidentally brings back.
```suggestion
* Fri Apr 19 2024 Tobias Brick <tobiasb@microsoft.com> - 1.1.1k-30
```

---

updated